packages/cli/src/commands/seed.ts

```diff
@@ -0,0 +1,498 @@
+    SimpleAllowListSchema,
+  ]),
+  incentives: z.array(
+    z.union([
+      AllowListIncentiveSchema,
+      ERC20IncentiveSchema,
+      ERC20VariableCriteriaIncentiveSchema,
+      ERC20VariableIncentiveSchema,
+    ]),
+  ),
+});
+
+async function getAllowList(
+  { allowList }: z.infer<typeof BoostSeedConfigSchema>,
+  { core }: { core: BoostCore },
+): Promise<AllowList> {
+  if (typeof allowList === 'string' && isAddress(allowList))
+    return await allowListFromAddress(
+      //@ts-expect-error i do what i want
+      { config: core._config, account: core._account },
+      allowList,
+    );
+  switch (allowList.type) {
+    case 'SimpleAllowList':
+      return core.SimpleAllowList(allowList as SimpleAllowListPayload);
+    case 'SimpleDenyList':
+      return core.SimpleDenyList(allowList as SimpleDenyListPayload);
+    default:
+      throw new Error('unusupported AllowList: ' + allowList);
+  }
+}
+
+async function fundBudget(
+  options: DeployableTestOptions,
+  erc20: MockERC20,
+  budget: Budget,
+  amount = parseEther('110'),
+) {
+  console.warn(`minting ${amount} to ${options.account.address}`);
+  await erc20.mint(options.account.address, amount);
+
+  await erc20.approve(budget.assertValidAddress(), amount);
+  await budget.allocate({
+    amount,
+    asset: erc20.assertValidAddress(),
+    target: options.account.address,
+  });
+
+  return { budget, erc20 };
+}
+
+export function makeSeed({
+  asset = '0xf3B2d0E4f2d8F453DBCc278b10e88b20d7f19f8D',
+  account = accounts[0].account,
+  chainId,
+}: {
+  asset?: Address;
+  account?: Address;
+  chainId: number;
+}): BoostConfig {
+  return {
+    asset,
+    protocolFee: 0n,
+    maxParticipants: 10n,
+    budget: {
+      type: 'ManagedBudget',
+      owner: account,
+      authorized: [account],
+      roles: [Roles.MANAGER],
+    },
+    action: {
+      type: 'EventAction',
+      actionClaimant: {
+        signatureType: SignatureType.FUNC,
+        signature: 'function mint(address to, uint256 amount)' as Hex,
+        fieldIndex: 0,
+        targetContract: zeroAddress,
+        chainid: chainId,
+      },
+      actionSteps: [
+        {
+          signature: 'event Minted(address to, uint256 amount)' as Hex,
+          signatureType: SignatureType.FUNC,
+          actionType: 0,
+          targetContract: zeroAddress,
+          chainid: chainId,
+          actionParameter: {
+            filterType: FilterType.EQUAL,
+            fieldType: PrimitiveType.ADDRESS,
+            fieldIndex: 0,
+            filterData: account,
+          },
+        },
+      ],
+    },
+    validator: {
+      type: 'SignerValidator',
+      signers: [account],
+      validatorCaller: account,
+    },
+    allowList: {
+      type: 'SimpleDenyList',
+      owner: account,
+      denied: [],
+    },
+    incentives: [
+      {
+        type: 'ERC20Incentive',
+        asset: asset,
+        strategy: 0,
+        reward: 1n,
+        limit: 1n,
+        manager: account,
```

Comment:
```suggestion
        manager: account,
        shouldMintAndAllocate: false
```

just to make this API obvious when they generate a seed